src/components/App.css

```diff
@@ -1,19 +1,77 @@
 .App {
-    display: grid;
-    grid-template-rows: 1fr auto 1fr;
-    height: 100vh;
+  display: grid;
+  grid-template-rows: 1fr auto 1fr;
+  grid-template-columns: 1fr;
+  height: 100vh;
+}
+
+.App--four {
+  grid-template-columns: 1fr 1fr;
 }
 
 .controls {
-    align-self: center;
-    justify-self: center;
-    display: grid;
-    justify-items: center;
-    grid-auto-flow: column;
+  align-self: center;
+  justify-self: center;
+  display: grid;
+  justify-items: center;
+  grid-column: 1 / span 2;
+  grid-auto-flow: column;
+}
+
+.App--four button div {
```

Comment:
we shouldn't directly style inner parts of another component by directly referencing its tags, let's instead add a possibility to add a custom class name as a prop on the Timer component, set a custom class on each used Timer in the App component and use the custom class name here instead of `button div`